dvc/path_info.py

```diff
@@ -18,8 +18,24 @@ def overlaps(self, other):
         return self.isin_or_eq(other) or other.isin(self)
 
     def isin_or_eq(self, other):
+        if isinstance(other, list):
+            others = other
+
+            for other in others:
+                if self.isin_or_eq(other):
+                    return True
+
+            return False
+
         return self == other or self.isin(other)  # pylint: disable=no-member
 
+    def isinside(self, others):
+        for other in others:
+            if other.isin(self):
+                return True
+
+        return False
```

Comment:
Likewise, this is not necessary, it does the same thing as `isin`, and there's cleaner ways to handle comparing against a list of `others`.

Reply:
Incorporated in https://github.com/iterative/dvc/pull/4938/commits/395167309234a0fb80cd66a416d3b52cc43f2e1e.